addons/html_editor/static/src/main/media/media_dialog/file_selector.js

```diff
@@ -129,6 +129,13 @@ export class FileSelectorControlPanel extends Component {
         this.debouncedValidateUrl = useDebounced(this.props.validateUrl, 500);
 
         this.fileInput = useRef("file-input");
+        const urlInputRef = useRef("urlInput");
+
+        useEffect(() => {
+            if (this.state.showUrlInput) {
+                urlInputRef.el.focus();
+            }
+        }, () => [this.state.showUrlInput]);
```

Comment:
I don't understand this:
- Either we somehow name the "search" input "url" input and this is weird?
- Either you want to focus the "Add URL" input and this is wrong?